notebooks/304-pytorch-post-training-quantization/304-pytorch-post-training-quantization.ipynb

```diff
@@ -0,0 +1,698 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "afa7e952",
+   "metadata": {},
+   "source": [
+    "# Post-Training Quantization of PyTorch models with NNCF\n",
+    "\n",
+    "This notebook is based on 'ImageNet training in PyTorch' example.\n",
+    "\n",
+    "The goal of this notebook is to demonstrate how to use the Neural Network Compression Framework NNCF 8-bit quantization in post-training mode (without the fine-tuning pipeline) to optimize a PyTorch model for the high-speed inference via OpenVINO Toolkit. The optimization process contains the following steps:\n",
+    "\n",
+    "1. Evaluate the original model\n",
+    "2. Transform the original model to a quantizaed one\n",
+    "3. Export optimized and original models to ONNX and then to OpenVINO IR\n",
+    "4. Compare perfomance of the obtained FP32 and INT8 models\n",
+    "\n",
+    "NOTE: This notebook requires C++"
+   ]
+  },
```

Comment:
Add a header `### Preparations`

Reply:
@vshampor Done

Reply:
@MaximProshin please provide an example or you can directly add the description where you feel it should be completed